tests/test_MainWindow.py

```diff
@@ -0,0 +1,163 @@
+        self.assertEqual(main_window_ui.undoVendorChangesButton.text(), "Undo Changes")
+
+    def test_select_vendor_button(self):
+        '''Test the defaults'''
+        self.assertEqual(main_window_ui.deselect_vendors_button_fetch.text(), "Deselect All")
+        self.assertEqual(main_window_ui.select_vendors_button_fetch.text(), "Select All")
+        self.assertEqual(main_window_ui.deselect_vendors_button_special.text(), "Deselect All")
+        self.assertEqual(main_window_ui.select_vendors_button_special.text(), "Select All")
+
+    def test_select_report_types_button(self):
+        '''Test the defaults'''
+        self.assertEqual(main_window_ui.deselect_report_types_button_fetch.text(), "Deselect All")
+        self.assertEqual(main_window_ui.select_report_types_button_fetch.text(), "Select All")
+        self.assertEqual(main_window_ui.toolButton.text(), "?")
+
+    def test_fetch_advanced_button(self):
+        '''Test the defaults'''
+        self.assertEqual(main_window_ui.fetch_advanced_button.text(), "Fetch Selected Reports")
+
+    def test_fetch_all_data_button(self):
+        '''Test the defaults'''
+        self.assertEqual(main_window_ui.fetch_all_data_button.text(), "Fetch All Reports")
+
+    def test_fetch_speical_data_button(self):
+        '''Test the defaults'''
+        self.assertEqual(main_window_ui.fetch_special_data_button.text(), "Fetch Special Report")
+
+    def test_select_file_button(self):
+        '''Test the defaults'''
+        self.assertEqual(main_window_ui.select_file_button.text(), "Select File")
+        self.assertEqual(main_window_ui.import_file_button.text(), "Import Selected File")
+
+    def test_search_button(self):
+        '''Test the defaults'''
+        self.assertEqual(main_window_ui.search_button.text(), "Search")
+
+    def test_setting_button(self):
+        '''Test the defaults'''
+        self.assertEqual(main_window_ui.concurrent_reports_help_button.text(), "?")
+        self.assertEqual(main_window_ui.concurrent_vendors_help_button.text(), "?")
+        self.assertEqual(main_window_ui.empty_cell_help_button.text(), "?")
+        self.assertEqual(main_window_ui.general_json_directory_button.text(), "Choose Folder...")
+        self.assertEqual(main_window_ui.general_json_directory_help_button.text(), "?")
+        self.assertEqual(main_window_ui.general_tsv_directory_help_button.text(), "?")
+        self.assertEqual(main_window_ui.general_tsv_directory_button.text(), "Choose Folder...")
+        self.assertEqual(main_window_ui.request_interval_help_button.text(), "?")
+        self.assertEqual(main_window_ui.special_json_directory_button.text(), "Choose Folder...")
+        self.assertEqual(main_window_ui.special_json_directory_help_button.text(), "?")
+        self.assertEqual(main_window_ui.special_tsv_directory_button.text(), "Choose Folder...")
+        self.assertEqual(main_window_ui.special_tsv_directory_help_button.text(), "?")
```

Comment:
general_json_directory tests can be removed
general_tsv_directory tests can be removed
special_json_directory tests can be removed
special_tsv_directory tests can be removed